CMakeLists.txt

```diff
@@ -628,6 +637,15 @@ if(NOT BUILD_HEADLESS)
     target_dependent_data_symlink_to_build(freeorion ${PROJECT_SOURCE_DIR}/default)
 endif()
 
+if(WIN32)
+    add_custom_command(TARGET freeorion POST_BUILD
+        COMMAND
+            ${CMAKE_COMMAND} -E copy_if_different "${CMAKE_PREFIX_PATH}/bin/python27.dll" "$<TARGET_FILE_DIR:freeorion>"
```

Comment:
Does `/bin/` get created during the CMake build? I don't think it's in the SDK by default...

Reply:
It is in the SDK by default, but the `bootstrap.bat` remove the `bin` directory, I have updated the `Build.md` instruction to tell people not to run `bootstrap.bat` when using CMake, for the reason of having an out-of-source build.